app/services/store_sp_metadata_in_session.rb

```diff
@@ -82,12 +44,12 @@ def update_session
       request_url: sp_request.url,
       request_id: sp_request.uuid,
       requested_attributes: sp_request.requested_attributes,
-      ial: ial_value,
-      ial2: ial2_value,
-      ialmax: ialmax_value,
-      aal_level_requested: aal_level_requested_value,
-      piv_cac_requested: piv_cac_requested_value,
-      phishing_resistant_requested: phishing_resistant_value,
+      ial: parsed_vot&.ial_value_requested,
```

Comment:
This value and the next couple succeeding ones are replaced in this PRs. I would not recommend changing them:

- https://github.com/18F/identity-idp/pull/10126
- https://github.com/18F/identity-idp/pull/10128
- https://github.com/18F/identity-idp/pull/10130

Additionally, on main, the values in these are no longer read.

Reply:
Removed.